todo.txt

```diff
@@ -0,0 +1,12 @@
+todo:
+
+remove all of the old UI summary code, replacing it with JS
```

Comment:
Is this strictly needed? Couldn't we just reuse the current Go implementation and call it from JS?

If nothing else it would save us time to get this released in v0.30.0.

Reply:
Unfortunately we need it... :disappointed: Otherwise there would be no way for the users to specify `handleSummary` and keep the current stdout text summary and output some other format to a file, for example. We can't keep the current hack of passing the Go function as a second argument, unless we want to maintain that indefinitely... :disappointed: